### PR TITLE
Minus Icon

### PR DESCRIPTION
**Pull request by drazenbuljovcic**
## Purpose

In the [design](https://www.figma.com/file/PSltEJWi5fJCJjBEihXbad/4---MVP---Acquisition-Journey?node-id=5304%3A28141) we have the need for a new icon, to represent the (-) sign.

image

## Approach and changes

Adding the new svg asset to the `icons` package.

## Definition of done

* [x] Development completed
* [x] Reviewers assigned
* [ ] Unit and integration tests
* [ ] Meets minimum browser support
* [ ] Meets accessibility requirements

### CODE REVIEW

File: packages/icons/README.md [hunk 1]
```diff
@@ -83,7 +83,20 @@ const SuccessMessage = ({ description }) => (
 );
 ```
 
-It is not possible to change the color of an external SVG using the `css` color property. Instead, you can use the [CSS `filter` hack](https://blog.union.io/code/2017/08/10/img-svg-fill/) to colorize the icon.
+#### Color your icon
```
Comment (connor-baer) [inline packages/icons/README.md:86]: What's the motivation behind these changes? 

When importing SVGs as files (as opposed to React components which is documented above) it's not possible to style them with CSS.

File: packages/icons/web/v1/minus_small.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path d="M9 7h6a1 1 0 1 1 0 2H1a1 1 0 0 1 0-2h8z" fill="#666"/>
```
Comment (connor-baer) [inline packages/icons/web/v1/minus_small.svg:2]: Monochromatic icons should use the `currentColor` value by conventions. This makes it easier to style them with CSS. 

```suggestion
    <path d="M9 7h6a1 1 0 1 1 0 2H1a1 1 0 0 1 0-2h8z" fill="currentColor"/>
```

**Review by connor-baer — CHANGES_REQUESTED** [review]

File: packages/icons/README.md (hunk 1 above)
Comment (drazenbuljovcic) [reply packages/icons/README.md:86]: Right, the color change is properly documented above!
I will revert this change, thanks for pointing out!

File: packages/icons/web/v1/minus_small.svg (hunk 2 above)
Comment (drazenbuljovcic) [reply packages/icons/web/v1/minus_small.svg:2]: Yep, I will update the asset file.

**Review by connor-baer — APPROVED** [review]
Thank you! 🙌🏻

**Review by connor-baer — CHANGES_REQUESTED** [review]
I just remembered one more thing: Can you [add a changeset](https://github.com/sumup-oss/circuit-ui/blob/main/CONTRIBUTING.md#submitting-a-pull-request), please?

**Review by connor-baer — APPROVED** [review]

File: .changeset/heavy-toys-thank.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/icons': major
```
Comment (connor-baer) [inline .changeset/heavy-toys-thank.md:2]:
```suggestion
'@sumup/icons': minor
```

File: .changeset/heavy-toys-thank.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+---
+
+Added a new asset to the icons package. The 'minus_small' icon.
```
Comment (connor-baer) [inline .changeset/heavy-toys-thank.md:5]: This will be added to the changelog for `@sumup/icons`, so the context is implied.

```suggestion
Added a new minus icon in a small size.
```

**Review by connor-baer — CHANGES_REQUESTED** [review]
> The only option that I had is to choose a major version for the icons package.

When `changesets` asks you which packages should receive a major bump, just select none and hit enter. It'll then ask you about minor and patch bumps. Alternatively, you can manually edit the markdown file.

> Also none of the other versions have been updated in the other projects that use this package.

`changesets` takes care of this when publishing a new version.

**Review by robinmetral — APPROVED** [review]
➖ (🙌)

**Review by connor-baer — APPROVED** [review]
